package.json

```diff
@@ -8,61 +8,54 @@
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "ng e2e",
-    "ws_copy_css": "mkdir -p dist/worldskills-angular-lib/css/ && cp -R ./projects/worldskills-angular-lib/src/lib/css/ ./dist/worldskills-angular-lib/css/",
```

Comment:
`ws_copy_css` no longer needed?

Reply:
The CSS Copy is no longer needed. we have planned to remove this for a while (finally got to it).

style.css was for the older layout we no longer use:
https://github.com/worldskills/worldskills-angular-lib/blob/master/projects/worldskills-angular-lib/src/lib/css/style.css

font.css maybe useful though. I'm going to look for a simpler way to include this.
https://github.com/worldskills/worldskills-angular-lib/blob/master/projects/worldskills-angular-lib/src/lib/css/font.css